src/i18n/en-US/makingARequest.ts

```diff
@@ -0,0 +1,25 @@
+const makingARequest = {
```

Comment:
Not sure I love this file/i18n name spacing, but I don't have any suggestions at the moment

Reply:
You're right, I'm seeing that we have just about all of the accessibility-related i18n in `accessibility.ts`.  This is inconsistent that we have a separate file for this view.  Not sure if it's better to split out the whole `accessibility.ts` or to just keep adding to it.  It's definitely getting a bit long.

Reply:
Dope!  This is not an accessibility page- never mind!!!